### PR TITLE
selectROI.py removed as samtools now can do it better and in parallel

### PR DESCRIPTION
**Pull request by szilvajuhos**
# nf-core/sarek pull request
## PR checklist

- [x] We do not need this obsolete script anymore as samtools can do it better and faster
- [ ] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by MaxUlysse** [comment]
Do we have a samtools process to do that?

**Comment by szilvajuhos** [comment]
`samtools view -@48 -L target.bed -o targeted.bam input.bam` can do that

**Comment by MaxUlysse** [comment]
ok, we'll think of a process to do that in DSL2 then

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Removed
 
+	[#234](https://github.com/nf-core/sarek/pull/243) - Removing obsolete script [#92]
```
Comment (MaxUlysse) [inline CHANGELOG.md:24]:
```suggestion
- [#234](https://github.com/nf-core/sarek/pull/243) - Removing obsolete script [#92]
```

**Comment by MaxUlysse** [comment]
> sure, sorry for the expandtab

no worry, there was just a markdown linting error

**Pull request merged**